### PR TITLE
changed fast forward time

### PR DESCRIPTION
**Pull request by vschs007**
fast forward time changed

### CODE REVIEW

File: js/px-video.js [hunk 1]
```diff
@@ -439,7 +439,7 @@ function InitPxVideo(options) {
 	obj.btnForward.addEventListener('click', function() {
 	    var targetTime = obj.movie.currentTime + obj.seekInterval;
 		if (targetTime > obj.movie.duration) {
-			obj.movie.currentTime = obj.movie.duration;
+			obj.movie.currentTime = 0;
```
Comment (hozefaj) [inline js/px-video.js]: this change looks incorrect. the logic is that if the `targetTime` exceed the movie duration, code goes to end of movie.

Why would you want to go to start of it?

Comment (vschs007) [inline js/px-video.js]: generally it happens when people want to see the end of video, and taking the user to the end of the video will end the video, I agree with your view that fetching initial value to time isn't either a good option,
But i have a idea on how taking user to just 2 or 3 secs before the end of video??

**Comment by hozefaj** [comment]
Looks like https://github.com/paypal/accessible-html5-video-player/pull/90/commits/5d2904ab75175db503df3998800af764a8a7fdff reverts the code...hence closing the PR.

**Comment by hozefaj** [comment]
@vschs007 Ideally look at what you-tube/netflix do for such cases. They are the gold standard for videos and this player should exactly follow what they do. Unless this player does something else, I don't see any reason for a change.

**Pull request closed without merging**